src/pages/index.tsx

```diff
@@ -24,16 +43,15 @@ export default function Home() {
 
   class Decks extends React.Component<DeckProps, DeckState> {
     render() {
-      const listItems = this.props.decks.map((deck) => (
-        <Link href={`/review/${deck.id}`}>
+      return this.props.decks.map((deck) => (
```

Comment:
Cool that you can return the value from `decks.map` and it fits right into the list below.